newsletter_laravel/app/Console/Commands/ExportNewsletterStories.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Services\NewsletterService;
+use Exception;
+use Illuminate\Console\Command;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use League\Csv\Writer;
+
+class ExportNewsletterStories extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'app:export-newsletter-stories';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Export all the stories with Newsletter review type';
+
+    public function __construct(NewsletterService $newsletterService)
+    {
+        parent::__construct();
+        $this->newsletterService = $newsletterService;
+    }
+
+    /**
+     * Executes the console command. Writes the needed data to a csv file
+     */
+    public function handle()
+    {
+        $io = new SymfonyStyle($this->input, $this->output);
+
+        try {
+            $data = $this->newsletterService->exportStories($io);
+            $csvData = $this->newsletterService->extractForCSV($data);
+
+            $csvFilename = 'newsletter_stories';
+            $csvPath = storage_path('app/' . $csvFilename);
+
+            $csv = Writer::createFromPath($csvPath, 'w+');
+            $csv->insertOne(['story_id', 'story_title', 'labels']);
+
+            foreach ($csvData as $rowData) {
+                $csv->insertOne($rowData);
+            }
+
+            $io->success('Data exported to CSV: ' . $csvPath);
```

Comment:
Laravel `Command` class has access to `$this->success` or `$this->error`, so you don't need to instantiate new `SymfonyStyle` first. Check Laravel's `InteractsWithIO.php` file